examples/interop/README.md

```diff
@@ -0,0 +1,98 @@
+### Description
+
+This folder contains example applications for mainchain and sidechain with POS consensus algorithm.
```

Comment:
Research team is working on SDK parameters recommended values and share what is ideal value for blockTime etc. maybe once that is finalized we can either update here or it will go into documentation.